rose/server/score.py

```diff
@@ -83,9 +99,11 @@ def process(players, track):
             elif player.x < config.matrix_width - 1:
                 player.x += 1
 
+        # Finally forget action
```

Comment:
This comment looks like a block comment for the next 2 lines, but is is a comment only for the first line. Adding a blank line between the next lines will make this more clear.

Reply:
done